packages/react-reconciler/src/ReactFiberTreeReflection.js

```diff
@@ -274,7 +276,15 @@ export function findCurrentHostFiber(parent: Fiber): Fiber | null {
 function findCurrentHostFiberImpl(node: Fiber) {
   // Next we'll drill down this component to find the first HostComponent/Text.
   if (node.tag === HostComponent || node.tag === HostText) {
-    return node;
+    if (
+      enableFloat &&
+      supportsHydration &&
+      isHydratableResource(node.type, node.memoizedProps)
+    ) {
+      // This is a crude opt out of Resources from this search algorithm
```

Comment:
Just leave this to finding the link tag for now. It'll be fixed once you add the new tag. These reflection helpers are semi-deprecated anyway.

Reply:
unfortunately, this errors if we don't do this

https://github.com/facebook/react/blob/main/packages/react-dom/src/client/ReactDOMRoot.js#L122-L132

Reply:
Should I just not check if the root was manipulated when enableFloat is on?

Reply:
Yea, that's not an important warning.

Reply:
Ok should be good to review